SU2_CFD/include/solvers/CFVMFlowSolverBase.inl

```diff
@@ -2535,6 +2559,10 @@ void CFVMFlowSolverBase<V, FlowRegime>::Friction_Forces(const CGeometry* geometr
   bool axisymmetric = config->GetAxisymmetric();
   bool roughwall = (config->GetnRoughWall() > 0);
   bool nemo = config->GetNEMOProblem();
+  bool wallfunctions=false;
+
+  for (iDim = 0; iDim < nDim; iDim++) RefVel2 += Velocity_Inf[iDim] * Velocity_Inf[iDim];
```

Comment:
```suggestion
RefVel2 = GeometryToolbox::SquaredNorm(nDim, Velocity_Inf);
```
Then you can prob also make it const su2double right there and get rid of the definition above. 
And you need to include the geometry_toolbox.hpp if not already there

Reply:
yes, I'm still getting used to using the toolbox :-) as well as getting used to not working with variable definitions at the top of the function instead of at the point where they are used for the first time.